src/business_logic/execution/objects.rs

```diff
@@ -208,6 +220,28 @@ impl Default for CallInfo {
                 builtin_instance_counter: HashMap::new(),
             },
             events: Vec::new(),
+            gas_consumed: 0,
+            failure_flag: false,
+        }
+    }
+}
+
+// ----------------------
+// CallResult structure
+// ----------------------
+
+pub struct CallResult {
+    pub gas_consumed: u64,
+    pub is_success: bool,
+    pub retdata: Vec<MaybeRelocatable>,
+}
+
+impl From<CallInfo> for CallResult {
+    fn from(info: CallInfo) -> Self {
+        Self {
+            gas_consumed: 0,
```

Comment:
why 0 as default for gas_consumed?

Reply:
i have to check if it have a default value, but i set it to 0 because is not actually used in the deprecated flow